module/VuFind/src/VuFind/Config/Upgrade.php

```diff
@@ -829,29 +830,53 @@ protected function upgradeReserves()
     }
 
     /**
-     * Upgrade EDS.ini.
+     * Upgrade EDS.ini
```

Comment:
Just to echo ThoWagen's suggestion, here's a convenient way to add the missing period for consistency.

```suggestion
     * Upgrade EDS.ini.
```